crm_create_project/views/crm_lead.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="crm_lead_view_form_crm_project_project" model="ir.ui.view">
+        <field name="name">CRM Lead Form Create Project</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="header" position="inside">
+                <button
+                    string="Create Project"
+                    name="%(crm_create_project.crm_create_project_action)d"
+                    type="action"
+                    class="oe_highlight"
+                    context="{'default_project_name': name, 'default_lead_id': id}"
+                    attrs="{'invisible': ['|', ('active','=',False), ('project_id', '!=', False)]}"
+                />
```

Comment:
issue: on runbot, there's a permissions issue.

![image](https://github.com/OCA/crm/assets/973709/3b62201a-08b4-427c-b435-2cdecdca1375)
![image](https://github.com/OCA/crm/assets/973709/c7b0990d-8f21-466b-82af-eb5a7c097bb9)
![image](https://github.com/OCA/crm/assets/973709/21592a88-cad8-4f58-bcdf-3849767fc8f6)

Reply:
The group shown in the image is not odoo core, it is from the crm_security_group module.
The permission for the wizard is for the group *Sales: User: Own Documents Only* which I think is the minimum group the project should be able to create. It could be for all groups but I think there should be a minimum restriction for the group *Sales: User: Own Documents Only*.
I don't think it's right to make this module depend on crm_security_group.

Reply:
Oh of course, I didn't realize that.